src/components/pages/login/LoginForm.jsx

```diff
@@ -19,12 +26,47 @@ export default function LoginForm() {
 
     // affichage
     return (
-        <form onSubmit={ handleSubmit }>
+        <LoginFormStyled onSubmit={ handleSubmit }>
             <h1>Bienvenue chez nous !</h1>
-            <br/>
+            <hr/>
             <h2>Connectez-vous</h2>
-            <input type="text" placeholder="Entrez votre prénom..." required value={ username } onChange={ handleChange }></input>
-            <button>Accédez à votre espace</button>
-        </form>
+            <Input value={ username } onChange={ handleChange } type="text" placeholder="Entrez votre prénom" Icon={<BsPersonCircle className="icon" />} required/>
```

Comment:
```suggestion
            <TextInput value={ username } onChange={ handleChange } type="text" placeholder="Entrez votre prénom" Icon={<BsPersonCircle className="icon" />} required/>
```

On va préférer TextInput car l'input peut etre de plusieurs type : radio, checkbox, textarea, etc.